pixeltable/tests/data/documents/layout-parser-paper.mmd

```diff
@@ -0,0 +1,268 @@
+# LayoutParser: A Unified Toolkit for Deep Learning Based Document Image Analysis
+
+Zejiang Shen
+
```

Comment:
What is `.mmd`? Is this another kind of document? How is this represented in the document types enum?